frontend/src/app/modules/auth/register-form/register-form.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { passwordMatchValidator } from '../validators/customValidator';
+
+@Component({
+  selector: 'app-register-form',
+  templateUrl: './register-form.component.html',
+  styleUrls: ['./register-form.component.scss']
+})
+export class RegisterFormComponent {
+
+  public registerForm: FormGroup;
+
+  public hidePass: Boolean = true;
+  public hideConfirmPass: Boolean = true;
+
+  constructor(
+    private formBuilder: FormBuilder
+  ) {
+    this.registerForm = this.formBuilder.group({
+      fullName: [, {
+        validators: [Validators.required,
+        Validators.pattern("^[a-zA-Z'][a-zA-Z-' ]+[a-zA-Z']?$")
+        ], updateOn: "change"
+      }],
```

Comment:
```suggestion
  constructor(private formBuilder: FormBuilder) {
    this.registerForm = this.formBuilder.group({
      fullName: {
        validators: [
        Validators.required,
        Validators.pattern("^[a-zA-Z'][a-zA-Z-' ]+[a-zA-Z']?$")
        ],
        updateOn: "change"
      },
```
Can you work on code style a bit? To make it look something like this. The same below